src/main/java/com/github/wovnio/wovnjava/WovnHttpServletResponse.java

```diff
@@ -16,11 +16,13 @@
     private PrintWriter writer;
     private ServletOutputStream output;
     private Headers headers;
+    private Utf8 unicodeConverter;
 
-    WovnHttpServletResponse(HttpServletResponse response, Headers headers) {
+    WovnHttpServletResponse(HttpServletResponse response, Headers headers, String encoding) {
         super(response);
         this.buff = new ByteArrayOutputStream();
         this.headers = headers;
+        this.unicodeConverter = new Utf8(encoding);
```

Comment:
you may as well promote this to a constructor parameter. No point passing in `encoding` just to construct `Utf8` out of it. I wouldn't be against renaming it to UnicodeConverter either.

Reply:
Good point.